src/helpers/sceneless-project.ts

```diff
@@ -2333,3 +2333,32 @@ export const createCompositor = async (
 
   return project
 }
+
+// In lieu of a versioning system, this can be used
+//  to update a ScenelessProject if its base structure changes.
+export const updateToLatest = async (_project: ScenelessProject) => {
+  const root = _project.scene.getRoot()
+  const coreProject = getProject(_project.id)
+
+  // Ensure audioContainer exists
+  const audioContainer = root.children.find((x) => x.props.id === 'audio')
+  if (!audioContainer) {
+    await coreProject.compositor.insert(
```

Comment:
If it's possible to insert at a specific index, let's insert at index 0 please. I know that this layer is invisible, but in case that gets broken somehow, it will at least then be rendering in the back, which is the next best thing I think